cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1021,6 +1021,12 @@ pub type Migrations = (
 	// unreleased
 	cumulus_pallet_xcmp_queue::migration::v4::MigrationToV4<Runtime>,
 	cumulus_pallet_xcmp_queue::migration::v5::MigrateV4ToV5<Runtime>,
+	// unreleased
+	pallet_assets::migration::next_asset_id::SetNextAssetId<
+		ConstU32<50_000_000>,
```

Comment:
How about defining and naming a const under `runtimes/constants/` instead of having this magic number here?

Reply:
it will be gone after the migration applied